colpali_engine/models/late_interaction/colpali/processing_colpali.py

```diff
@@ -0,0 +1,67 @@
+from __future__ import annotations
+
+from typing import List, Optional
+
+from PIL import Image
+from transformers import BatchFeature, PaliGemmaProcessor
+
+from colpali_engine.utils.processing_utils import BaseVisualRetrieverProcessor
+
+
+class ColPaliProcessor(BaseVisualRetrieverProcessor, PaliGemmaProcessor):
+    def __init__(self):
+        BaseVisualRetrieverProcessor.__init__(self)
```

Comment:
is that the way to do it with double inheritance ? 
isn't there a way to disambiguate the super().__init__ by specifying the class ?

Reply:
Actually I don't need the `BaseVisualRetrieverProcessor` init, so we're good now!